jina/__init__.py

```diff
@@ -204,5 +208,8 @@ def _set_nofile(nofile_atleast=4096):
 from jina.serve.executors import BaseExecutor as Executor
 from jina.serve.executors.decorators import monitor, requests
 
+# Custom Gateway
+from jina.serve.gateway import BaseGateway as Gateway
```

Comment:
Shouldn't we just rename BaseGateway to Gateway ? I am not sure to understand why two names

Reply:
We're trying to mimic BaseExecutor